docs/source/api/apollo-gateway.mdx

```diff
@@ -100,11 +100,18 @@ const gateway = new ApolloGateway({
 
 ###### `supergraphSdl`
 
-`String`
+`string | SupergraphSdlHook | SupergraphSdlObject`
 </td>
 <td>
 
-A [supergraph schema](../#federated-schemas) ([generated with the Rover CLI](https://www.apollographql.com/docs/rover/supergraphs/#composing-a-supergraph-schema)) that's composed from your subgraph schemas. The supergraph schema includes directives that specify routing information for each subgraph.
+When `supergraphSdl` is a `string`: A [supergraph schema](../#federated-schemas) ([generated with the Rover CLI](https://www.apollographql.com/docs/rover/supergraphs/#composing-a-supergraph-schema)) that's composed from your subgraph schemas. The supergraph schema includes directives that specify routing information for each subgraph.
+
+When `supergraphSdl` is a `SupergraphSdlHook`: An `async` function that returns an object containing a `supergraphSdl` string as well as a `cleanup` function. The hook accepts an object containing 3 properties:
+1. `update`: A function which updates the supergraph schema
```

Comment:
Make this an unordered list (all lists are unordered unless they describe a sequence of steps)

"A function which" -> "A function that" in all three cases